CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Changelog
 
+# [0.14.0] - 2022-01-05
+
+### Added
+
+- Web: Add removeLinks and getLinks method
+- iOS/Android: Code block bindings
+
+### Fixed
+
+- Common: Code block fixes
```

Comment:
Actually, these should be "Code block & quote bindings/fixes".

Reply:
Fixed

Reply:
I actually meant 'Code block & quote bindings' in 'Added' and 'Code block & quote fixes' in 'Fixed', sorry 😅

Reply:
The bindings are only for iOS/Android, the fixes are common (they're in the Rust model).

Reply:
Okay, I fixed it

Reply:
Thanks! And sorry for the misunderstanding!